src/main/java/org/intellij/sdk/codesync/clients/ClientUtils.java

```diff
@@ -74,6 +74,19 @@ private static HttpPost getHttpPost(String url, JSONObject JSONPayload, String a
         return httpPost;
     }
 
+    private static HttpPost getHttpPost(String url, String accessToken) throws InvalidJsonError {
```

Comment:
Instead of creating a new variant, I think you can use the previous one and pass in an empty payload?

e.g. `new JSONPayload ()`